contributing/code/tests.rst

```diff
@@ -24,6 +24,11 @@ tests, such as Doctrine, Twig and Monolog. To do so,
 
     $ composer update
 
+.. tip::
+
+    You need to be on ``master`` or version branch (e.g. ``5.1``). Otherwise some dependencies
```

Comment:
I would show the usage of the `COMPOSER_ROOT_VERSION` env var instead like this:

```
COMPOSER_ROOT_VERSION=5.1.x-dev symfony composer update
```

Reply:
I didn't know it works like that actually :smile:

Reply:
@xabbux I changed the tip accordingly. Is the wording ok?

Reply:
I wonder if we shouldn't simply always make use of the env var and simply add a sentence like "replace 5.1 with the Symfony version your patch targets" or so.